### ISSUE TITLE
Win8 WDK x64 dbghelp.dll not found

### ISSUE THREAD

**Issue by derekbruening** [DynamoRIO/drmemory#1080]
_From [bruen...@google.com](https://code.google.com/u/109494838902877177630/) on November 05, 2012 20:09:54_

my DRi#894/drmemi#1078 changes to how dbghelp is found break the locating
of the x64 dbghelp.dll from the Win8 WDK.

on my laptop which has Win8 WDK but not Win7 WDK, for an x64 build:
CMake Error at CMakeLists.txt:439 (message):^M
  dbghelp.dll required and not found^M

kraken finds:
-- Using C:/WINDDK/7600.16385.1//tools/tracing/amd64/dbghelp.dll

we want this:
/c/Program Files (x86)/Windows Kits/8.0/Debuggers/x64/dbghelp.dll

but PROGRAMW6432 is the 64-bit /c/Program Files

_Original issue: http://code.google.com/p/drmemory/issues/detail?id=1080_

**Comment by bailehang** [comment]
I found the dbghelp.dll in these path:
```
C:\Windows\SysWOW64\
C:\Windows\System32\
C:\Windows\winsxs\amd64_........
C:\Windows\winsxs\x86_.....
```
and I have a question:tools/DRstats/CMakeList.txt:116  need uuid.lib. Where is it? windows 7 system [6.1.7601] not found. Do I need to install windows kit?

**Comment by derekbruening** [comment]
I believe this dbghelp issue was fixed in dbb9bae616b0de99daa5fc7c40dbe76638640426

Re: uuid.lib: it looks like DRstats has some hardcoded paths which should be generalized.  Yes, a Windows SDK does need to be installed in general.

**Comment by derekbruening** [comment]
Please note that many of use are still using older Visual Studio installations and have not really tested new installations very well, so we appreciate any contributions to update the build infrastructure to work with newer toolchains.
